common/source/docs/common-future-wiki-changes.rst

```diff
@@ -471,11 +471,25 @@ For landing in fixed wing, manual throttle controlled modes, there is an ``RCx_O
 
 On Tailsitters page, under Vectored Thrust:
 -------------------------------------------
-
+gitk
```

Comment:
unintentional?

Reply:
the joys of automatic keyboard and mouse switching between machines

Reply:
:) At least it didn't go out as an email
I've started doing "git diff" before every commit... also helps to catch unintentional whitespace changes

Reply:
added notes about motor mask and q_options for copter TS without control surfaces...